src/types/app-client.ts

```diff
@@ -260,9 +261,9 @@ export interface FundAppAccountParams {
 /** Source maps for an Algorand app */
 export interface AppSourceMaps {
   /** The source map of the approval program */
-  approvalSourceMap: SourceMapExport
+  approvalSourceMap: algosdk.ProgramSourceMap
```

Comment:
This would be extending reliance on algosdk types externally which I'm not sure we want to do.

Reply:
Ah good catch. Didn't realize the SDK used a class